openmetadata-service/src/test/java/org/openmetadata/service/util/FullyQualifiedNameTest.java

```diff
@@ -52,9 +52,9 @@ void test_quoteName() {
     assertEquals("\"a.b\"", FullyQualifiedName.quoteName("\"a.b\"")); // Leave existing valid quotes
     assertEquals("a", FullyQualifiedName.quoteName("\"a\"")); // Remove quotes when not needed
     // we now allow quotes
-    assertEquals("\\\"a", FullyQualifiedName.quoteName("\"a"));
-    assertEquals("a\\\"", FullyQualifiedName.quoteName("a\""));
-    assertEquals("a\\\"b", FullyQualifiedName.quoteName("a\"b"));
+    assertEquals("\"a", FullyQualifiedName.quoteName("\"a"));
```

Comment:
why are we changing these tests?

Reply:
@sonika-shah are we removing these tests because its handling the inner double quote, what happens if the name does contain a \" ?